qatrack/qatrack_core/tasks.py

```
@@ -92,35 +92,54 @@ def run_periodic_scheduler(model, log_name, handler, time_field="time", recurren
+                target_end = end_today + timezone.timedelta(days=day_offset)
+
+                occurences = getattr(instance, recurrence_field).between(target_start, target_end, inc=True)
+                
+                logger.info(
+                    "Occurences for %s %s: %s (between %s & %s)" % (
+                        model._meta.model_name,
+                        instance.id,
+                        occurences,
+                        target_start,
+                        target_end,
+                    )
```

Ruff (0.15.17)

[warning] Use format specifiers instead of percent format

Replace with format specifiers

(UP031)